pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -685,8 +687,13 @@ def sort_legal_provision(elem):
         """
 
         sort_title = elem['Title'] if 'Title' in elem else ''
-        sort_number = elem['OfficialNumber'] if 'OfficialNumber' in elem else None
-        return sort_title, sort_number
+        if 'OfficialNumber' in elem and elem['OfficialNumber']:
+            sort_number_type = 0
+            sort_number = elem['OfficialNumber']
+        else:
+            sort_number_type = 1
+            sort_number = ''
+        return sort_title, sort_number_type, sort_number
```

Comment:
what do you think about:
```
        sort_title = elem.get('Title', '')
        sort_number = elem.get('OfficialNumber', '')
        # the additional sorting key below must be produced so that records without OfficialNumber
        # are sorted after records with OfficialNumber
        sort_empty_official_number_last = ( len(sort_number) == 0 )
        return sort_title, sort_empty_official_number_last, sort_number
```

Reply:
Good suggestion. Changes already committed.